src/components/backup/BackupConfirmPasswordStep.js

```diff
@@ -183,9 +184,12 @@ const BackupConfirmPasswordStep = () => {
         Alert.alert('Error while trying to backup');
       },
       password,
-      walletId: params?.walletId,
+      walletId,
     });
-  }, [dispatch, latestBackup, params?.walletId, password, walletCloudBackup]);
+    setTimeout(() => {
```

Comment:
walletCloudBackup will not throw but will call the onError method that is passed in, which means it could fail, call the onError, and then call this setTimeout to say that backup was successful

Reply:
perhaps pass in an onSuccess as well

Reply:
https://github.com/rainbow-me/rainbow/pull/1003